lib/jsdom/level2/languages/javascript.js

```diff
@@ -7,7 +7,9 @@ exports.javascript = function (element, code, filename) {
 
   if (window) {
     try {
+      window.document._currentScript = element;
       vm.runInContext(code, window, { filename: filename });
+      window.document._currentScript = undefined;
```

Comment:
Should be null both before and after. (To take care of `before`, we need to set it in the Document constructor.